README.md

```diff
@@ -52,6 +52,7 @@ This will use settings from different sources in the following order of priority
 
 System properties are preferred over environment variables. The following system properties & environment variables can be used for configuration:
 
+* `kubeconfig` / `KUBECONFIG`: Name of the kubernetes config file to read (defaults to `~/.kube/config`)
```

Comment:
Sorry, I'm not sure but isn't the property name `KUBERNETES_KUBECONFIG_FILE`? Have you tried configuring `kubeconfig` using this property?

Reply:
Haven't tried it, but checked the code. This looks like a legacy property that doesn't match the other naming conventions:
https://github.com/fabric8io/kubernetes-client/blob/6a47d262e37bee71b9579551a9a56288761f02c2/kubernetes-client/src/main/java/io/fabric8/kubernetes/client/Config.java#L110

So yes, the Java variable has `KUBERNETES_KUBECONFIG_FILE` this name, but not its  value neither the conversion to environment variables.

Reply:
oh, yeah. You're right.